Notifier-service/src/app.ts

```diff
@@ -22,6 +23,7 @@ export class App {
 
     constructor(private container:AppContainer){
         this.app = express();
+        this.app.use(cors());
```

Comment:
* CORS takes configuration information right. regarding which ip's to allow for incoming request. Maybe you can add configuration entry for that information in .env , and load that here with the cors.

Reply:
@rajatkb Is it ok?